Pract. 02/Solutions/Task10.cpp

```diff
@@ -0,0 +1,43 @@
+﻿// Task10.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+using namespace std;
+
+int main()
+{
+    int firstNumber, secondNumber;
+    char operation;
+    cout << "Enter first number: ";
+    cin >> firstNumber;
+    cout << "Enter an operation: ";
+    cin >> operation;
+    cout << "Enter second number: ";
+    cin >> secondNumber;
+    switch (operation)
+    {
+    case '+': cout << firstNumber + secondNumber; break;
+    case '-': cout << firstNumber - secondNumber; break;
+    case '*': cout << firstNumber * secondNumber; break;
+    case '/': {
+        if (secondNumber != 0) {
+            cout << firstNumber / (double)secondNumber;
+        }
+        else {
+            cout << "Second number cannot be zero";
+        }
+    }
```

Comment:
What will happen if you have one more case?

Reply:
I wont have more cases. Its either 0 or not 0. In other task i would do it with else if for the 0 value and else for the other one but here is not needed.